Source/WebCore/loader/FTPDirectoryParser.cpp

```diff
@@ -146,7 +146,9 @@ FTPEntryType parseOneFTPLine(std::span<LChar> line, ListState& state, ListResult
                                 pos++;
                             if (pos < linelen && line[pos] == ',') {
                                 unsigned long long seconds = 0;
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
                                 sscanf(byteCast<char>(p.subspan(1)).data(), "%llu", &seconds);
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
```

Comment:
This source is supposed to be dead code.  Seems like it's still compiled, but should be removable.

Reply:
@beidson @cdumez Can we delete this code?